build.sbt

```diff
@@ -406,13 +439,29 @@ lazy val tests = crossProject
       "databaseClasspath" -> classDirectory.in(semanticdbIntegration, Compile).value.getAbsolutePath
     ),
     buildInfoPackage := "scala.meta.tests",
-    libraryDependencies += "org.scalatest" %%% "scalatest" % "3.0.1" % "test"
+    libraryDependencies += "org.scalatest" %%% "scalatest" % "3.2.0-SNAP10" % "test"
   )
   .jvmConfigure(_.dependsOn(testkit, interactive, metac))
+  .jvmSettings(
+    // TODO: Workaround for what seems to be a bug in ScalaTest 3.2.0-SNAP10.
+    // Without adding scalacheck to library dependencies, we get the following error:
+    // > testsJVM/test
+    // [info] Compiling 79 Scala sources to /Users/eburmako/Projects/scalameta/tests/jvm/target/scala-2.12/test-classes...
+    // [trace] Stack trace suppressed: run last testsJVM/test:executeTests for the full output.
+    // [error] (testsJVM/test:executeTests) java.lang.NoClassDefFoundError: org/scalacheck/Test$TestCallback
+    // [error] Total time: 19 s, completed Feb 1, 2018 3:12:34 PM
+    libraryDependencies += "org.scalacheck" %% "scalacheck" % "1.13.5" % "test"
```

Comment:
@bvenners This only happens in ScalaTest 3.2.0-SNAP10, but not in 3.2.0-SNAP6. Looks like a bug?

Reply:
are we missing scalacheck on our classpath?

Reply:
We don't have a dependency on scalacheck, so yeah it's not on the classpath.